src/XrdAccSciTokens.cc

```diff
@@ -666,22 +733,48 @@ class XrdAccSciTokens : public XrdAccAuthorize
     uint64_t m_next_clean{0};
     XrdSysError m_log;
     AuthzBehavior m_authz_behavior{AuthzBehavior::PASSTHROUGH};
+    std::string m_cfg_file;
 
     static constexpr uint64_t m_expiry_secs = 60;
 };
 
+std::string      cfgSciTokens;
+
 extern "C" {
 
-XrdAccAuthorize *XrdAccAuthorizeObject(XrdSysLogger *lp,
-                                       const char   *cfn,
-                                       const char   *parm)
```

Comment:
You mentioned that you would like to use this for end-stage authorization (i.e. not pushed). If so, you can simply leave this function in and it's body would be
return XrdAccAuthorizeObjectAdd(lp, cfn, parm, 0);

Of course, it would need to be defined after XrdAccAuthorizeObjectAdd or you would need an ugly extern.

Reply:
I'm a bit confused.  I thought it had to be one way or the other.  If we include this function, then it will continue to work in the current method, and the new (pushed) way?